models/org.go

```diff
@@ -453,6 +455,8 @@ func (org *User) getUserTeams(e Engine, userID int64, cols ...string) ([]*Team,
 	return teams, e.Where("team_user.org_id = ?", org.ID).
 		And("team_user.uid = ?", userID).
 		Join("INNER", "team_user", "team_user.team_id = team.id").
+		Join("INNER", "user", `"user".id=team_user.uid`).
```

Comment:
XORM will use ` as the uniform quote and will replace it as database's self quote. So just use `,

```Go
Join("INNER", "user", "`user`.id=team_user.uid").
```